blogposts/announcing-sourcegraph-3.3.md

```diff
@@ -1,44 +1,217 @@
 
-<!-- 
-Each section should have a screencast when possible, usually without sound as they only for 30s max.
+  [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes](https://github.com/sourcegraph/deploy-sourcegraph)
 
-  <p class="container">
-    <div style="padding:56.25% 0 0 0;position:relative;">
-      <iframe src="https://player.vimeo.com/video/$VIMEO_VIDEO_ID?color=0CB6F4&amp;title=0&amp;byline=0&amp;portrait=0&autoplay=1&loop=1" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
-    </div>
-    <p style="text-align: center"><a href="https://vimeo.com/VIMEO_VIDEO_ID" target="_blank">View on Vimeo</a></p>
-  </p>
--->
+Want to try pre-release development builds? Change the Docker image tag to  `sourcegraph:insiders`.
+
+</div>
+
+---
+
+This release focusses on making Sourcegraph easier to set up, configure and manage repositories.
+
+We’ve completely overhauled how repositories are selected and kept in-sync, benefiting everyone, but especially organizations with thousands (or tens of thousands) of repositories. We’ve also added Sentry to our list of third party developer tool integrations and improved Swift code intelligence and navigation.
+
+Read on for the details, and thanks to our customers and community for reporting issues and providing feedback. You’re helping to make each Sourcegraph release the best one yet!
+
+**Release highlights**:
+
+[**🛠️ New! Config based repository selection**](#repo-config)<br />
+Repositories now chosen via explicit lists and query rules in the external service config.
+
+**[🔌 New Sentry integration](#sentry) and [Bitbucket Server fixes](#bitbucket-fixes)**<br />
+Jump to Sentry from Sourcegraph and other integrations improvements.
+
+[**🧭 More precise code intelligence**](#code-intel)<br />
+Symbols scoped to the current file and improved Swift support.
+
+[**🚢  Deployments secure by default with TLS pre-configured**](#deployments)<br />
+TLS on all Terraform plans and improved AWS deployment docs.
+
+[**Making Sourcegraph a part of your dev tools stack**](#grok-sourcegraph)<br />📣
+Tips for integrating Sourcegraph and code search into your team for better code.
+
+[**📖 New docs design**](#docs)<br />
+Better UI, new page navigation, and docs improvements for site admins.
+
+Also in this release:
+
+[**📝 Changelog**]()<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**]()<br />
+Sourcegraph couldn’t be what it is without the community
+
+<div id="repo-config"></div>
+
+## New repository configuration for GitHub, GitLab, and Bitbucket Server
+
+Prior to 3.3, integrating with a code host and selecting which repositories to use was a manual process:
+
+![](/blog/old-repository-sync.png)
```

Comment:
broken image

Reply:
It's not broken when rendered by Gatsby.